Manifests, problem with parallel builds

I've tried out the new [manifests](https://github.com/microsoft/vcpkg/blob/master/docs/specifications/manifests.md) feature with an existing multi-project solution that has been successfully using `vcpkg` for CI. The `vcpkg.json` and msbuild configuration to build and install the dependencies worked properly but there's a problem with parallel builds.

When building in Visual Studio or on the command line with `msbuild /m` there are numerous `vcpkg` step failures. From the logs below it seem there are multiple instances of the command attempting to lock ` C:\Tools\vcpkg-btc\.vcpkg-root`. Build on the command line without the `/m` option removes this error.

```
  22>VcpkgInstallManifestDependencies:
         Failed to take the filesystem lock on C:\Tools\vcpkg-btc\.vcpkg-root:
             The requested resource is in use.
    22>c:\Tools\vcpkg-btc\scripts\buildsystems\msbuild\vcpkg.targets(102,5): error MSB3073: The command ""c:\Tools\vcpkg-btc\v
       cpkg.exe" install --triplet "x64-windows-static" --vcpkg-root "c:\Tools\vcpkg-btc\\" "--x-manifest-root=c:\Dev\github\s
       ipsorcery_bitcoin\build_msvc\\" "--x-install-root=c:\Dev\github\sipsorcery_bitcoin\build_msvc\vcpkg_installed\\" --bina
       rycaching" exited with code 1. [c:\Dev\github\sipsorcery_bitcoin\build_msvc\libbitcoin_qt\libbitcoin_qt.vcxproj]
    22>Done Building Project "c:\Dev\github\sipsorcery_bitcoin\build_msvc\libbitcoin_qt\libbitcoin_qt.vcxproj" (default target
       s) -- FAILED.
    12>c:\Tools\vcpkg-btc\scripts\buildsystems\msbuild\vcpkg.targets(102,5): error MSB3073: The command ""c:\Tools\vcpkg-btc\v
       cpkg.exe" install --triplet "x64-windows-static" --vcpkg-root "c:\Tools\vcpkg-btc\\" "--x-manifest-root=c:\Dev\github\s
       ipsorcery_bitcoin\build_msvc\\" "--x-install-root=c:\Dev\github\sipsorcery_bitcoin\build_msvc\vcpkg_installed\\" --bina
       rycaching" exited with code 1. [c:\Dev\github\sipsorcery_bitcoin\build_msvc\libbitcoin_zmq\libbitcoin_zmq.vcxproj]
    12>Done Building Project "c:\Dev\github\sipsorcery_bitcoin\build_msvc\libbitcoin_zmq\libbitcoin_zmq.vcxproj" (default targ
       ets) -- FAILED.
```

--- a/scripts/buildsystems/msbuild/vcpkg.targets
+++ b/scripts/buildsystems/msbuild/vcpkg.targets
@@ -99,7 +99,7 @@
 
   <Target Name="VcpkgInstallManifestDependencies" Condition="'$(VcpkgEnabled)' == 'true' and '$(VcpkgEnableManifest)' == 'true' and '$(VcpkgManifestInstall)' == 'true'" BeforeTargets="ClCompile">
     <Message Text="Installing vcpkg dependencies" Importance="High" />
-    <Exec Command="%22$(VcpkgRoot)vcpkg.exe%22 install --triplet %22$(VcpkgTriplet)%22 --vcpkg-root %22$(VcpkgRoot)\%22 %22--x-manifest-root=$(VcpkgManifestRoot)\%22 %22--x-install-root=$(VcpkgInstalledDir)\%22 --binarycaching" StandardOutputImportance="High" />
+    <Exec Command="%22$(VcpkgRoot)vcpkg.exe%22 install --x-wait-for-lock --triplet %22$(VcpkgTriplet)%22 --vcpkg-root %22$(VcpkgRoot)\%22 %22--x-manifest-root=$(VcpkgManifestRoot)\%22 %22--x-install-root=$(VcpkgInstalledDir)\%22" StandardOutputImportance="High" />
   </Target>
 
   <Target Name="AppLocalFromInstalled" AfterTargets="CopyFilesToOutputDirectory" BeforeTargets="CopyLocalFilesOutputGroup;RegisterOutput" Condition="'$(VcpkgEnabled)' == 'true' and '$(VcpkgApplocalDeps)' == 'true'">

